src/Packagist/WebBundle/HealthCheck/MetadataDirCheck.php

```diff
@@ -20,7 +20,7 @@ public function __construct(array $awsMetadata)
 
     public static function isMetadataStoreMounted(array $awsMeta): bool
     {
-        if ($awsMeta['primary'] && $awsMeta['has_instance_store']) {
+        if (isset($awsMeta['primary']) && isset($awsMeta['has_instance_store'])) {
```

Comment:
this is wrong. It won't check the boolean values of these params, which will break the prod (a secondary node would get handled as primary, as `isset($awsMeta['primary'])` would return `true` for `['primary' => false]`

Reply:
You are right !

Reply:
what you should do, to be consistent with other places using `$awsMeta` in the codebase is to change that to `!empty($awsMeta) && $awsMeta['primary'] && ...`